playbooks/provision.yml

```diff
@@ -12,6 +12,10 @@
       become_user: root
       tags: security
 
+    - role: oefenweb.swapfile
+      become: yes
+      tags: swapfile
+
```

Comment:
Can we use this role conditionally so that it doesn't install it's dependencies unless it's being used? So using `when: swapfile_size != false` or similar.

Reply:
Great idea. I added that.